haas/rest.py

```diff
@@ -164,10 +170,13 @@ def foo(bar, baz, quux):
           whose second is an integer (the status code).
     """
     def register(f):
+
+        methods = method if isinstance(method, list) else [method]
```

Comment:
Also, I want to point out: the presence of this change threw me for a bit, since it has nothing to do with the stated purpose of this pr. Then I remembered the context in which we decided we needed this and see why you added it. I'm fine leaving it here, but please try to be more clear about the contents of a pr in its description in the future.